.travis.yml

```diff
@@ -9,11 +9,14 @@ python:
 env:
   - DJANGO="django>=1.11,<1.12"
```

Comment:
Your research indicates `Django 1.11` should be dropped.

Reply:
Actually, I was wrong, it's `Dropped support for Python 2.6 and 3.3, and Django < 1.11`, so this will work. :smile: